### ISSUE TITLE
Duplicate Time Validator

### ISSUE THREAD

**Issue by zzzsz** [digicert/pkilint#106]
The function of `time.TimeCorrectEncodingValidator()` in `create_validity_validator_container()` is equal to `time.UtcTimeCorrectSyntaxValidator()` plus` time.GeneralizedTimeCorrectSyntaxValidator()`.
And in the end, these three are all in `doc_validator` of crl.

```
def create_validity_validator_container(additional_validators=None):
    if additional_validators is None:
        additional_validators = []
    return validation.ValidatorContainer(
        validators=[
                       crl_validity.CrlSaneValidityPeriodValidator(),
                       time.TimeCorrectEncodingValidator(),
                   ] + additional_validators,
        path='certificateList.tbsCertList'
    )
```

**Comment by CBonnell** [comment]
Thanks, I think the `ValidatorContainer`'s path should be scoped to `certificateList.tbsCertList.thisUpdate` and `certificateList.tbsCertList.nextUpdate` to ensure that `GeneralizedTime` values appearing CRL extensions are not incorrectly flagged if they contain a year between 1950 and 2049.

Also consider having the `TimeCorrectEncodingValidator` merely return upon encountering an invalid syntax value instead of raising a finding that will be duplicated by the Syntax validators.
